addon/components/rdf-input-fields/concept-scheme-radio-buttons/edit.js

```diff
@@ -0,0 +1,71 @@
+import { action } from '@ember/object';
+import { guidFor } from '@ember/object/internals';
+
+import SimpleInputFieldComponent from '../simple-value-input-field';
+import { triplesForPath, updateSimpleFormValue} from '@lblod/submission-form-helpers';
+import { SKOS } from '@lblod/submission-form-helpers';
+import rdflib from 'browser-rdflib';
+
+
+function byLabel(a, b) {
+  const textA = a.label.toUpperCase();
+  const textB = b.label.toUpperCase();
+  return (textA < textB) ? -1 : (textA > textB) ? 1 : 0;
+}
+
+export default class RdfInputFieldsconceptSchemeRadioButtonsEditComponent extends SimpleInputFieldComponent {
```

Comment:
Kudos for extending from `SimpleInputFieldComponent`! 
What made you do so? I'm assuming to make use of `nodeValue`?

Reply:
Codes comes from concept-scheme-selector as idea is pretty similar to radio-buttons: "multiple options to select from and only 1 can get selected at a time." So assumed that radio-buttons would have same flow , my assumption was right  apparently :D